docs/api/puppeteer.page.exposefunction.md

```diff
@@ -8,6 +8,12 @@ The method adds a function called `name` on the page's `window` object. When cal
 
 If the puppeteerFunction returns a `Promise`, it will be awaited.
 
+:::caution
+
+The function will only be installed on the main frame and not child frames.
```

Comment:
Is this correct? We do `Page.addScriptToEvaluateOnNewDocument` and `evaluate` on each current frame

Reply:
Right, probably not completely accurate but addScriptToEvaluateOnNewDocument does not work for OOPIFs, only called for local frames.